app/services/idv/profile_from_applicant.rb

```diff
@@ -3,11 +3,11 @@ class ProfileFromApplicant
     attr_reader :profile
 
     def self.create(applicant:, user:, normalized_applicant:)
-      profile = Profile.new(user: user)
+      profile = Profile.new(user: user, deactivation_reason: :verification_pending)
       plain_pii = pii_from_applicant(applicant, normalized_applicant)
       profile.encrypt_pii(user.user_access_key, plain_pii)
       profile.save!
-      profile
+      [profile, plain_pii]
```

Comment:
would it make sense to have `profile` and `plain_pii` methods on this service class instead? Returning an array from a `create` method seems like something that could get us into trouble / be unclear. 

Alternatively, having a hash with well-named keys would be an improvement over an array. In that case, I do think we would still be exhibiting [primitive obsession](https://sourcemaking.com/refactoring/smells/primitive-obsession)